### ISSUE TITLE
Window capture : top corners are wrongly cut

### ISSUE THREAD

**Issue by InfoLibre** [linuxmint/muffin#572]
```
 * Cinnamon version 4.6.7
No PPA
 * Mint 20 Cinnamon
 * 64 bit
 ```

**Issue**
When I take a screenshot of a window with Cinnamon Screenshot, the two top corners are wrongly cut in the resulting file.

![96354688-ae1cc200-10d9-11eb-8507-8dc27cb36b6f](https://user-images.githubusercontent.com/3955499/96354755-66e30100-10da-11eb-90d1-1dadf76026b6.png)

**Steps to reproduce**
Open a Terminal window (Administration - Terminal in Mint Menu). Take a screenshot of this window with Cinnamon Screenshot.

**Expected behaviour**
The two top corners of the picture should be round, with transparency. The colour of the background shouldn't appear around these corners.
Pictures created with Cinnamon Screenshot have a transparency layer but it seems that it isn't enough, there's another problem in the corners.

**Comment by InfoLibre** [comment]
Can you re-open my bug report until it is solved, please ? Or is it unuseful ?

**Comment by leigh123linux** [comment]
@InfoLibre We need to know what the last muffin/cinnamon version that worked properly so we can bisect.

**Comment by InfoLibre** [comment]
When I take a screenshot using Alt + PrtScn, the bug is the same. It's not a Gnome Screenshot bug but a bug with Windows management.

**Comment by leigh123linux** [comment]
Can you stop posting garbage!
If you don't  provide last working version info I will close the issue.

**Comment by InfoLibre** [comment]
You ask me "Have you tested the tools in another DE?" and you say it's garbage when I answer to you ??????

**Comment by leigh123linux** [comment]
> It sure is in Mint. The question though is, is the issue specific to Mint or is it inherited by Ubuntu / Debian, etc.
> 
> Also:
> Speaking of older Mint versions, which were fine: which Mint version exactly? Are you sure you used the same theme in those older Mint versions?

You have ignored this



> @InfoLibre We need to know what the last muffin/cinnamon version that worked properly so we can bisect.

and this

**Comment by leigh123linux** [comment]
> When I take a screenshot using Alt + PrtScn, the bug is the same. It's not a Gnome Screenshot bug but a bug with Windows management.

Alt + PrtScn is the **gnome-screenshot** keybinding

**Comment by InfoLibre** [comment]
You have ignored this -> no, yesterday, it took several hours to me to download old ISO and to test it and I did not finished my tests. But I will not lost more time with people that first close tickets without explanation and after insult me. I will use Linux Mint witout make reports, as everybody. And stop to make donations. Bye.

**Comment by InfoLibre** [comment]
Solved with Linux Mint Cinnamon 20.3.

![Capture d’écran du 2022-01-07 09-10-03](https://user-images.githubusercontent.com/3955499/148513281-b676a32f-6d2a-4121-8d51-55161efd1aac.png)
